app-backend/migrations/src/main/scala/migrations/51.scala

```diff
@@ -0,0 +1 @@
+../../../../src_migrations/main/scala/50.scala
```

Comment:
This needs to point to 51.scala, or 52 if you choose to bump your migrations up a number